react-props-and-expressions/my-app/src/App.js

```diff
@@ -0,0 +1,22 @@
+// import logo from './logo.svg';
+import './App.css';
+
+function CustomButton({text, color}) {
```

Comment:
It would be better to extract the CustomButton component into a separate file and import it from the main App component. However, for this exercise, no big deal.